app/models/taxon_change.rb

```diff
@@ -126,7 +128,15 @@ def active_children_conflict?
     end
     return true
   end
-    
+  
+  def is_branching?
+    taxon_change_taxa.map( &:taxon_id ).include? taxon_id
```

Comment:
Since only a `TaxonSplit` can be "branching" can you move this to TaxonSplit? Putting it in `TaxonChange` means *all* taxon changes will respond to this, even thought `is_branching?` is meaningless for a `TaxonMerge`.

Also, I tend to find it more readable when we use the `input_taxa` and `output_taxa` methods, e.g. `output_taxa.map(&:id).include?( input_taxon.id )`. It does require loading those associates which is a bit of a performance burn, but I think they're probably already loaded anyway in most cases where this method would be called.